configure/templates/default/www.conf.erb

```diff
@@ -380,6 +380,7 @@ env[DD_PROFILING_ENABLED] = <%= (node[:datadog][:enable_profiling] || 0) && "tru
 env[DD_TRACE_ENABLED] = <%= (node[:datadog][:enable_trace_agent] || 0) && "true" || "false" %>
 env[DD_AGENT_HOST] = $HOSTNAME
 env[DD_SERVICE] = <%= node.read("datadog", "tags", "service") || "dev" %>
+env[DD_VERSION] = <%= (node["commit_hash"]) || "env[DD_VERSION] = lazy {node[commit_hash]}" %>
```

Comment:
this code is a bit suspect
You want to use `lazy` inside cookbooks/configure/recipes/php.rb instead:
```
variables(
    lazy {
      node["php"]["fpm"].merge({
         commit_hash: node.run_state[:jbx_version]
      })
   }
```

then in the template you can do 
```
<%= @commit_hash %>
```

Reply:
Well, see, I didn't know I could do this with a `merge`, that's a pro-tip! 👏 

> ```
> variables(
>     lazy {
>       node["php"]["fpm"].merge({
>          commit_hash: node.run_state[:jbx_version]
>       })
>    }
> )
> ```